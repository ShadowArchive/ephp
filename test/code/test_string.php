<?php

$a = "data";
print "index 1 from 'data' => [" . $a[1] . "]\n";

var_dump($a["a"][0.1][10]);

$b = ["data"];

var_dump($b[0]["a"][0.1][10]);

