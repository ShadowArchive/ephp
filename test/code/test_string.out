index 1 from 'data' => [a]

Warning: Illegal string offset 'a' in {{CWD}}/test/code/test_string.php on line 6

Notice: String offset cast occurred in {{CWD}}/test/code/test_string.php on line 6

Notice: Uninitialized string offset: 10 in {{CWD}}/test/code/test_string.php on line 6
string(0) ""

Warning: Illegal string offset 'a' in {{CWD}}/test/code/test_string.php on line 10

Notice: String offset cast occurred in {{CWD}}/test/code/test_string.php on line 10

Notice: Uninitialized string offset: 10 in {{CWD}}/test/code/test_string.php on line 10
string(0) ""
